packages/account/src/Sections/Security/ConnectedApps/connected-apps.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { DesktopWrapper, MobileWrapper, Button, Modal, Icon, DataTable, DataList } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import ErrorComponent from 'Components/error-component';
+import { WS } from 'Services/ws-methods';
+import getConnectedAppsColumnsTemplate from './data-table-template';
+
+class ConnectedApps extends React.Component {
```

Comment:
function? 😅

Reply:
can you explain a little more?
if you mean I should use functional component instead of a class component: I know we can use functional and react hook here but, managing hook here is a little harder here and I personally prefer to use `state` here but, if there is some point behind that, I can change it to a functional component. :pray:

Reply:
what parts makes hooks harder to use here compared to a class component?

Reasons for hooks:
- We can move logic into small reusable and isolated units - which has the benefits of reduced duplication, reduced component size (in lines of code). Instead of repeating fetching, loading, handling error in each component this could be a hook that is shared.
- Compatibility with new library updates - libraries are dropping support for class components - e.g. mobx-react
- Using hooks is more aligned to how react works (top --> down data flow), and will make it easier to reason and write components once you get used to it.

But if you wrote the component before we introduced hooks I guess it's fine to continue with it as it is?

Reply:
@oskar-binary Thanks for the explanation :pray: 
yes, you are right we can continue.